src/App.tsx

```diff
@@ -12,7 +12,7 @@ const AppContainer = styled.div`
   flex-direction: column;
 `;
 
-function App() {
+export function App(): JSX.Element {
```

Comment:
it would be nice to write App following the same pattern as the others

`export const App: React.FC = () => `